packages/Tag/src/Tag.styles.js

```diff
@@ -146,15 +85,96 @@ export const Delete = styled(Button.Close)(({ size }) => {
     height: ${width[size]};
     justify-content: center;
     min-height: auto;
-    padding: ${padding[size]};
     width: ${width[size]};
 
-    &:hover {
-      background: ${tokens.color.blackLighten60};
-    }
-
     &:focus {
       background: ${tokens.color.blackLighten60};
     }
   `;
 });
+
+const lightBackground = css`
+  ${Delete} {
+    color: ${tokens.color.blackLighten70};
+    &:hover {
+      color: ${tokens.color.blackLighten60};
+    }
+  }
+`;
+
+const darkBackground = css`
```

Comment:
Maybe renaming these two constants to `DeleteButtonStylesForDarkBackground/LightBackground` ?